web/app/.eslintrc

```diff
@@ -85,6 +85,6 @@
     // These rules represent best practices that we're not adhering to, and should be enabled
     "jsx-a11y/click-events-have-key-events": 0,
     "no-param-reassign": 0,
-    "react/no-did-update-set-state": 0
+    "react/no-did-update-set-state": 1
```

Comment:
You can just delete this line, since the rule is enabled by default.

Reply:
Thanks for the feedback @klingerf . Ill make the change and push